### PR TITLE
6 create objects

### LINKED ISSUE #6: Create objects

### PR DESCRIPTION
**Pull request by MAChristiansen**
closes #6 , #4 and #5

### CODE REVIEW

File: StringR.Store/Model/Order.swift [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+struct Order {
+
+    var orderId: String
+    var customerId: String
+    var stringerId: String
+    var racketType: Racket.RacketType
+    var tensionVertical: Double
+    var tensionHorizontal: Double
+    var stringId: String
+    var deliveryDate: Double
```
Comment (Jaafar92) [inline StringR.Store/Model/Order.swift]:
```suggestion
    var deliveryDate: Long
```

Should this not be a Long? And same with the ones below 😀

File: StringR.Store/Model/Economy.swift [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+struct Economy {
+
+    var tennisStrung: Int
+    var badmintonStrung: Int
+    var squashStrung: Int
+    var revenue: Double
+    var purchase: Double
+    var profit: Double
+    var unpaid: Double
+}
```
Comment (Jaafar92) [inline StringR.Store/Model/Economy.swift:21]: Should we also add 'Expenses' ?

File: StringR.Store/Model/Racket.swift [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+//  Created by Marcus Christiansen on 13/09/2019.
+//  Copyright © 2019 StringR. All rights reserved.
+//
+
+import Foundation
+
+struct Racket {
+
+    var racketId: String
+    var brand: String
```
Comment (Jaafar92) [inline StringR.Store/Model/Racket.swift]: Not an Enum? 😀

File: StringR.Store/Model/Racket.swift [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    var modelName: String
+    var weight: Int
+    var main: Int
+    var cross: Int
+
+    enum RacketType: String {
+        case TENNIS = "Tennis"
+        case BADMINTON = "Badminton"
+        case SQUASH = "Squash"
+    }
```
Comment (Jaafar92) [inline StringR.Store/Model/Racket.swift]: Is it possible to have the enum in a file for it self? That way we can use if for other purposes than 'Racket', eg. when showing economy.

File: StringR.Store/Model/RacketString.swift [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+        case ORANGE = "Orange"
+        case PINK = "Pink"
+        case PURPLE = "Puple"
+        case RED = "Red"
+        case SILVER = "Silver"
+        case TEAL = "Teal"
+        case WHITE = "White"
+        case YELLOW = "Yellow"
+    }
+}
```
Comment (Jaafar92) [inline StringR.Store/Model/RacketString.swift:24]: Same this these Enums? :-)

File: StringR.Store/Model/Shop.swift [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+import Foundation
+
+struct Shop {
+
+    var shopId: String
+    var address: Address
+    var phoneNumber: String
+    var storageId: String
+    var teamId: String
+    var conomyId: String
```
Comment (Jaafar92) [inline StringR.Store/Model/Shop.swift]:
```suggestion
    var companyId: String
```

**Review by Jaafar92 — COMMENTED** [review]
Looks really great 👍 There is some minor things that we might be able to improve.

File: StringR.Store/Model/Racket.swift (hunk 4 above)
Comment (MAChristiansen) [reply StringR.Store/Model/Racket.swift]: You can use them every where just type: "Racket.RacketType" ;)

**Pull request merged**